samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -1959,6 +1961,29 @@ public boolean getHasAttachment(){
     return hasAttachment;
   }
 
+  public boolean getShowFeedbackLink() {
+    return (getActionString() == "takeAssessment"
```

Comment:
StringUtils.isEqualsAny(getActionString(), TAKE_ASSESSMENT, TAKE_ASSESSMENT_VIA_URL, PREVIEW_ASSESSMENT).

These should be constants, probably are already created in a different class.

Reply:
There are constants defined, but as int
```
  // delivery action
  public static final int TAKE_ASSESSMENT = 1;
  public static final int PREVIEW_ASSESSMENT = 2;
  public static final int REVIEW_ASSESSMENT = 3;
  public static final int GRADE_ASSESSMENT = 4;
  public static final int TAKE_ASSESSMENT_VIA_URL = 5;
```